etc/kube/start-minikube-vm.sh

```diff
@@ -123,9 +123,7 @@ set -x
 # Kill pachctl port-forward and kubectl proxy
 killall kubectl || true
 
-if [[ "${DEPLOY_FLAGS}" = "--storage-v2" ]]; then
-	# Port forward to postgres
-	POSTGRES_POD=$(kubectl get pod -l suite=pachyderm,app=postgres -o jsonpath="{.items[].metadata.name}")
-	export POSTGRES_POD
-	kubectl port-forward "$POSTGRES_POD" 32228:5432 &
-fi
+# Port forward to postgres
+POSTGRES_POD=$(kubectl get pod -l suite=pachyderm,app=postgres -o jsonpath="{.items[].metadata.name}")
+export POSTGRES_POD
+kubectl port-forward "$POSTGRES_POD" 32228:5432 &
```

Comment:
There is a script `./etc/kube/forward_postgres.sh` that does this, but uses the service, instead of a single pod.